src/index.js

```diff
@@ -108,11 +108,16 @@ if (window.CanvasRenderingContext2D) {
     }
   }
 
+
   loadCustomCSS(state.advanced.customCSS);
 
   logDebugInfo(state.advanced);
 
   fullAPIReady.then(() => loadConfig()).then(() => {
+    if (preloadWorker) {
+      initTransports();
```

Comment:
Hmm why do we define a function for this? Can we move the function body into this `if` block instead?
Can you also move `const { preloadWorker } = state.advanced` here? Just to collocate better.

Reply:
Actually I think having a function is fine here since the code below are a bunch of functions. Please ignore my previous comment. I think this PR is good for merging now. @mparizeau-pdftron just need your approval.